runelite-client/src/main/java/net/runelite/client/plugins/wasdcamera/WASDCameraListener.java

```diff
@@ -95,11 +95,26 @@ else if (config.right().matches(e))
 					case KeyEvent.VK_SLASH:
 						// refocus chatbox
 						plugin.setTyping(true);
-						clientThread.invoke(() ->
-						{
-							plugin.unlockChat();
-						});
+						clientThread.invoke(() -> plugin.unlockChat());
 						break;
+					case KeyEvent.VK_1:
+					case KeyEvent.VK_2:
+					case KeyEvent.VK_3:
+					case KeyEvent.VK_4:
+					case KeyEvent.VK_5:
+					case KeyEvent.VK_6:
+					case KeyEvent.VK_7:
+					case KeyEvent.VK_8:
+					case KeyEvent.VK_9:
+					case KeyEvent.VK_0:
+						if (config.numbers())
+						{
+							final int newKeyCode = e.getKeyCode() == KeyEvent.VK_0
+								? KeyEvent.VK_F10
+								: e.getKeyCode() + 0x40 - 1; // This is the difference between num key
```

Comment:
Shouldn't this be `e.getKeyCode() + 0x3f`? Also, this could probably be defined as a constant:

```
private static final int NUMBERS_F_KEY_DIFFERENCE = KeyEvent.VK_F1 - KeyEvent.VK_1;
```